### PR TITLE
rfc0016: indexed

### PR DESCRIPTION
**Pull request by rjbs**

### CODE REVIEW

**Comment by haarg** [comment]
There's a number of places in this that still say `kv` rather than `indexed`.

File: rfc0016.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+```perl
+for my ($i, $value) ($alpha->{beta}->[0]->%[ keys $alpha->{beta}->[0]->@* ]) {
+  say "$i == $value";
+}
+```
+
+This is pretty bad.
+
+With `kv`, we write this:
```
Comment (Leont) [inline rfc0016.md:44]: I presume this `kv` should also be `indexed`

**Review by oodler577 — APPROVED** [review]
This is well done. I hope this (and n-at-a-time foreach) opens the door for a complementary "interleaved"/"zip"/"meshed" keyword in core rather than in List::Utils, etc.

**Comment by neilb** [comment]
> In list context, `indexed LIST` evalutes to a list twice the size of the array

I think that should be "twice the size of LIST".

**Pull request merged**